src/router/routes.ts

```diff
@@ -4,7 +4,23 @@ const routes: RouteRecordRaw[] = [
   {
     path: '/',
     component: () => import('layouts/MainLayout.vue'),
-    children: [{ path: '', component: () => import('pages/Index.vue') }],
+    children: [
+      { path: 'login', component: () => import('pages/login.vue') },
```

Comment:
If you navigate to the KebunKitani app now, its showing a blank screen. Because its not going to login by default (path should be blank for default). Should login page be the first screen to show or the dashboard? You decide.